### DISCUSSION TITLE
Can I include this installer in Commercial app？

### DISCUSSION THREAD

**Discussion by Sinsjr2** [dotnet/windowsdesktop#2879] [Q&A] [answered]
### Question
I know This repository is The Mit License and this installer ( https://dotnet.microsoft.com/ja-jp/download/dotnet/6.0 ) is multi license,
Can I include this installer in commercial app installer CD?

"dotnet windows" includes Licenses other than The MIT License.
https://github.com/dotnet/core/blob/main/license-information-windows.md
Also, "THIRD-PARTY-NOTICES.txt" in install folder is described many OSS.

As above this installer is multi license, so I'm wondering if it can be inluded in commercial app installer CD.

**Comment by richlander** [comment, marked as answer]
Yes. The main question you want to answer is whether .NET 6 gives you redistributable rights. It does, both for Windows and other supported operating systems.

**Reply by Sinsjr2** [reply]
The MIT LICENSE needs to show license indication in principle, 
so do I have to show license indication ?

If I need to to show license indication,
Should I place this license file in my CD as follows ?
https://github.com/dotnet/windowsdesktop/blob/main/LICENSE

```
CD/
├── LICENSE
├── MyApp.exe
└── windowsdesktop-runtime-6.0.5-win-x64.exe
```

**Reply by richlander** [reply]
The license should be for your app not just the framework. Within that, I'd suggest linking to [this file](https://github.com/dotnet/core/blob/main/license-information-windows.md) and simply saying that "other licenses from Microsoft also apply."

**Reply by Sinsjr2** [reply]
Thank you for your kind reply.

I will follow your suggestion, I will add this project and "License information for .NET on Windows" into  license indication on which my app depends.
